index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="#" method="POST">
+
+          <ul class="form-content">
+            <li> <label><input type="text" name="fullname" placeholder="Full name"
```

Comment:
- [ ] Please kindly update your placeholder's styling so they match the design.  You can do so by making use of the pseudo-class `::placeholder`.  You can get info about it [here](https://www.w3schools.com/cssref/sel_placeholder.asp)

Your document will look stunning.   ✔️